source/Outfit.cpp

```diff
@@ -181,20 +181,12 @@ void Outfit::Load(const DataNode &node)
 			mass = child.Value(1);
 		else if(child.Token(0) == "licenses" && (child.HasChildren() || child.Size() >= 2))
 		{
-			auto isNewLicense = [](const vector<string> &c, const string &val) noexcept -> bool {
-				return find(c.begin(), c.end(), val) == c.end();
-			};
 			// Add any new licenses that were specified "inline".
 			if(child.Size() >= 2)
-			{
-				for(auto it = ++begin(child.Tokens()); it != end(child.Tokens()); ++it)
-					if(isNewLicense(licenses, *it))
-						licenses.push_back(*it);
-			}
+				licenses.insert(child.Tokens().begin() + 1, child.Tokens().end());
 			// Add any new licenses that were specified as an indented list.
 			for(const DataNode &grand : child)
-				if(isNewLicense(licenses, grand.Token(0)))
-					licenses.push_back(grand.Token(0));
+				licenses.insert(grand.Token(0));
```

Comment:
Mind implementing `clear` and `remove` support? #3907

Reply:
Not `clear` though right? Isn't that syntax deprecated?

Also

```
remove licenses
    Navy
```

or

```
licenses
    remove Navy
```

Reply:
the second one ?
there isn't any ship that requires more than one license in the game though so clear doesn't really feel like it is needed to me

Reply:
Up until very recently we did have ships (navy in particular) that required both a "Navy" license and "carrier" license. (following EVN's style of multiple licenses). There's nothing wrong with that method, and plenty of strengths for doing so. As such, having support for managing multiple licenses for a ship is something that aught to be maintained.